utils/build.ps1

```diff
@@ -723,37 +841,66 @@ function Build-CMakeProject {
       } else {
         TryAdd-KeyValue $Defines CMAKE_Swift_COMPILER (Join-Path -Path (Get-PinnedToolchainTool) -ChildPath  "swiftc.exe")
       }
-      TryAdd-KeyValue $Defines CMAKE_Swift_COMPILER_TARGET $Arch.LLVMTarget
+      if (-not ($Platform -eq "Windows")) {
+        TryAdd-KeyValue $Defines CMAKE_Swift_COMPILER_WORKS = "YES"
```

Comment:
Curious, does CMake assume NO otherwise?

Reply:
The default is check (run some tests).